pennylane/operation.py

```diff
@@ -2346,7 +2345,7 @@ def eigvals(self):
         standard_observables = {"PauliX", "PauliY", "PauliZ", "Hadamard"}
 
         # observable should be Z^{\otimes n}
-        self._eigvals_cache = pauli_eigs(len(self.wires))
+        self._eigvals_cache = qml.pauli.pauli_eigs(len(self.wires))
```

Comment:
Note that `Tensor` is getting removed by @lillian542 , so there will be a merge conflict.